io.github.tofudd.lanchat.terminal.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
```

Comment:
This file should be ideally maintained in your repository and not on Flathub; this way you can update it as part of the release workflow.